Cue/app/tabs/library/LibraryHome.js

```diff
@@ -76,10 +80,27 @@ class LibraryHome extends React.Component {
   componentWillReceiveProps(newProps: Props) {
     this.setState({
       refreshing: false
-    })
+    })
+
+    if (newProps.inaccessibleDecks && newProps.inaccessibleDecks.length > 0) {
+      newProps.inaccessibleDecks.forEach(deck => {
+        Alert.alert(
+          (Platform.OS === 'android' ?
+            'The deck “' + deck.name + '” is no longer available from the original owner' :
+            'The Deck “' + deck.name + '” Is No Longer Available From The Original Owner'),

```

Comment:
`The Deck “' + deck.name + '” Is No Longer Available *from the* Original Owner`

Can you also test on decks with moderately-long titles to ensure you don't run into #152? If you need to save words, I'd get rid of "`The Deck`" from the beginning since it's mostly understood from context.